src/Core/Managed/Shared/Extensibility/Implementation/TaskTimer.cs

```diff
@@ -66,7 +66,11 @@ public void Start(Func<Task> elapsed)
 
             TaskEx.Delay(this.Delay, newTokenSource.Token)
                 .ContinueWith(
-                    async previousTask => 
+#if !NET40
+                async previousTask =>
+#else
+                    previousTask =>
```

Comment:
Re line 78 below:
This is why VAR is so bad. I am reading that (unchanged) bit of code and I need to stop and think about the type returned by elapsed(). I have to look around and rely on correctly used conventions for a correct guess. :)